backend/src/main/resources/db-scripts/migration/V1.8__create_document_office_user_group_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS
+  documentation_office_user_group (
+    id UUID NOT NULL PRIMARY KEY,
+    user_group_path_name VARCHAR(255) UNIQUE NOT NULL,
+    documentation_office_id UUID NOT NULL,
+    is_internal BOOL NOT NULL
+    -- It is not possible to reference the incremental_migration schema from the ris-backend public schema
+    -- CONSTRAINT fk_documentation_office_id FOREIGN KEY(documentation_office_id) REFERENCES incremental_migration.documentation_office(id)
+  );
```

Comment:
We will want to have a foreign key constraint on the new field "userGroup" on the procedure entity, as we want to have a Cascading Delete (remove reference). The procedure entity lives in the incremental_migration schema. -> Should we move the user group to the migration schema? :-(